js/components/graph/FocusBar.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import ReactModal from 'react-modal';
+import PropTypes from "prop-types";
+import Focus from "./Focus";
+import * as focusInfo from "./sidebar/focus_descriptions.js";
+
+// These lists are in reverse order to what ends up appearing on the screen
+const computerScienceFocusData = [
+    ["web", "Web Technologies"],
+    ["theory", "Theory of Computation"],
+    ["HCI", "HumanComp Interaction"],
+    ["game", "Video Games"],
+    ["systems", "Computer Systems"],
+    ["vision", "Computer Vision"],
+    ["NLP", "Computational Linguistics"],
+    ["AI", "Artificial Intelligence"],
+    ["sci", "Scientific Computing"],
+  ];
+
+const COLORS = ["#d472c7", "#ce73d3", "#c974d2", "#c672d2", "#bd71d3", "#b575d5", "#b072d3", "#ab72d3", "#a172d2"];
```

Comment:
I know that `linear-gradient()` would work for the actual `.focus` tabs themselves, but it does not allow me to carry over their background color to the `border-color` of `::after`, which is why I used this setup. I tried looking for a fix (one idea was using `mix-blend-mode`) but that did not really work.
I couldn't find a way to inherit colors cross-property without the variables, or using transparency (which doesn't work with my current design of using ::after since that already relies on transparency).

Reply:
@leenlab2 Very interesting research! Ultimately I'm still not happy with this approach, but also don't want this to hold up the rest of the PR. What I want you to do is *remove* the code related to colour gradients for the tabs from this branch, and instead set a uniform background colour for each tab, with a different colour for selected tabs, and on hover.

Then after this PR has been merged in, we can revisit adding a colour gradient.

Reply:
alright!

Reply:
Should I still try to increase the height on selection? It's a bit complicated since I've been using `display: inline-block` to make them appear stacked from right to left. When of the tabs changes height, it raises the entire menu bar up to fit.

Reply:
Ah somehow I thought you were already doing this, but no you don't have to in this PR. Let's file that away as something to improve in a future branch.